src/nvim/option.c

```diff
@@ -598,23 +598,20 @@ void set_init_1(void)
     }
   }
 
-  /*
-   * 'maxmemtot' and 'maxmem' may have to be adjusted for available memory
-   */
+  // 'maxmemtot' and 'maxmem' are ignored in Neovim, but we set them to the
+  // amount of available memory.
```

Comment:
Sure, `tenc` was made hidden because it was common to set it in vimrc:s, but I would guess this option is less common, so should be OK to remove.